spark/src/main/scala/io/delta/sharing/spark/DeltaSharingClient.scala

```diff
@@ -109,12 +109,12 @@ private[sharing] class DeltaSharingRestClient(
     val target = getTargetUrl("shares")
     val shares = ArrayBuffer[Share]()
     var response = getJson[ListSharesResponse](target)
-    shares ++= response.items
+    if (response != null && response.items != null) shares ++= response.items
```

Comment:
I'm not familiar with the code style here, but I'd prefer to always surround with `{` ... `}` for if-block. cc @zsxwing

Reply:
Sure, I added the {} for the block.